modules/UI/audio_levels/AudioLevels.js

```diff
@@ -6,48 +6,6 @@ import UIUtil from '../util/UIUtil';
  * Responsible for drawing audio levels.
  */
 const AudioLevels = {
-    /**
-     * Fills the dot(s) with the specified "index", with as much opacity as
-     * indicated by "opacity".
-     *
-     * @param {string} elementID the parent audio indicator span element
-     * @param {number} index the index of the dots to fill, where 0 indicates
-     * the middle dot and the following increments point toward the
-     * corresponding pair of dots.
-     * @param {number} opacity the opacity to set for the specified dot.
-     */
-    _setDotLevel(elementID, index, opacity) {
-        let audioSpan
```

Comment:
This was not used?

Reply:
At least I wasn't able to find any usage of it...